src/uwtools/tests/drivers/test_driver.py

```diff
@@ -68,7 +68,10 @@ class ConcreteDriverCycleLeadtimeBased(Common, driver.DriverCycleLeadtimeBased):
 
 
 class ConcreteDriverTimeInvariant(Common, driver.DriverTimeInvariant):
-    pass
+
+    @property
+    def output(self):
+        return {"foo": "/path/to/foo", "bar": ["/path/to/bar1", "/path/to/bar2"]}
```

Comment:
So that the `show_output()` task method can be realistically tested, override the default, exception-raising `output` inherited from `Driver` for this test class.